src/libs/actions/Report.js

```diff
@@ -1152,10 +1152,11 @@ function navigateToConciergeChat() {
  * @param {Object} policy
  * @param {String} reportName
  * @param {String} visibility
+ * @param {Array} policyMembers
  */
-function addPolicyReport(policy, reportName, visibility) {
+function addPolicyReport(policy, reportName, visibility, policyMembers) {
```

Comment:
i think `policy` should probably become `policyID` 

```suggestion
function addPolicyReport(policyID, reportName, visibility, policyMembers) {
```

and then we update any spot it was `policy` as well.